### ISSUE TITLE
Outdated Todoist API

### ISSUE THREAD

**Issue by jimbrig** [AlexAsplund/PSTodoist#1]
```powershell
Error: Invoke-RestMethod: C:\Users\jimbr\OneDrive\Documents\PowerShell\Modules\PSTodoist\Public\Get-TodoistTask.ps1:37
$Request = Invoke-RestMethod -Headers $Header -Method Get -Uri 'http …
| Please use https://api.todoist.com/rest/v1/ endpoint. 
| See https://developer.todoist.com/rest/v1/ for details.   
```

**Comment by Wishmaster172** [comment]
Yes, I'm facing the exact same issue. :(

**Comment by AlexAsplund** [comment]
I don't use todoist anymore so I can't try it out, but i did a quick search/replace and pushed it. Feel free to submit a PR if it doesn't work.

**Comment by jrobiii** [comment]
Hey @AlexAsplund, only tangentially related.  You said that you no longer use Todoist... What have you replaced it with (asking for a friend :-)?

**Comment by AlexAsplund** [comment]
@jrobiii 
Hey!

I use a messy mix right now:

- Microsoft Todo/Outlook tasks and or calendars (multiple accounts)
- Jira/Miro etc.
- Fiancé 😄

I see different customers as totally different/separate contexts so I use their ecosystem of tools. The only sync I have is a custom calendar sync I made to my main calendar using a mix of powershell and Power Automate.

**Comment by jrobiii** [comment]
Thanks for the reply and all I can say is wow!  That is an outstanding
number of tools. I was waffling between outlook tasks and ms todo for a
while. And then I found obsidian tasks.  It is so cool to create a task
right where you take notes and then use a Tasks query to gather everything
in one place.

Lately I noticed that I spend a significant amount of time messing around
with due dates and priorities to get tasks in the right order.  Really I
need to be able to just reorder the tasks like in Todoist where you just
drag them around. Unfortunately that's not the way obsidian tasks work.

I thought about trying to do something to sync tasks between obsidian and
Todoist and the just manage tasks in Todoist. That's how I ended up looking
at PSTodoist (I sort of gravitate to PowerShell).

Thanks for the peek into your task workflow.  My adventure continues.
